src/lib/krb5/krb/get_in_tkt.c

```diff
@@ -1881,7 +1863,9 @@ k5_populate_gic_opt(krb5_context context, krb5_get_init_creds_opt **out,
         if (retval)
             goto cleanup;
         if (creds->times.starttime) starttime = creds->times.starttime;
-        krb5_get_init_creds_opt_set_tkt_life(opt, creds->times.endtime - starttime);
+        krb5_get_init_creds_opt_set_tkt_life(opt,
```

Comment:
I agree this is awkward.  I don't like putting a newline right after the open paren of a function call, but it might make sense to introduce a local to hold ``ts_delta(cred->times.endtime, starttime)``.

(Fundamentally I wish the krb5_get_init_creds_opt_ functions had used krb5_gic_opt as a prefix so they weren't so long, but that ship sailed long ago.)